### DISCUSSION TITLE
c++_shared STL

### DISCUSSION THREAD

**Discussion by michael-burrowes-ts** [premake/premake-core#1806] [Q&A]
I'm using premake to generate Visual Studio projects that compile with Clang, and I need to use c++_shared for my stl rather than c++_static. It doesn't appear from the doc, but is there a way to specify this in my .lua file?

**Comment by samsinsane** [comment]
I don't believe there's a way to do that, to my knowledge our VS Clang support goes as far as emitting `ClangCL` for the platform toolset. We would welcome a PR if you wanted to add support for specifying the STL.
